src/Widgets/FastView/FastList.vala

```diff
@@ -64,6 +64,11 @@ public class Noise.FastView : Gtk.TreeView {
     }
 
     construct {
+        columns = new Gee.ArrayList<Type> ();
+        foreach (var type in ListColumn.get_all ()) {
```

Comment:
Any reason why you moved this from GenericList?

Reply:
Yes, before, this constructor got called before creating `GenericList`s, but the columns were empty. If i left it in the other constructor, I would had to duplicate some code from here (like creating the `FastModel`). (I hope my explanation was clear).

Reply:
It's not entirely clear, to be honest.

Reply:
Okay, I'll try to tell it differently. Before, when you were creating a `GenericList` instance, it started by calling the parent class (`FastView`) construct block, before calling the GenericList's construct block itself.

The FastView's construct block was using `columns`, because when we were using C#-style construction, we could just pass them from GenericList with `base` after having initialized them. But after changing code to use GObject construction style, it was `null`, and it caused a lot of problems.

I hope it was clearer this time (I'm sorry, my English isn't perfect, I can't say everything I would like to).

Reply:
Your explanation was right the first time around. I did not realize `GenericList` is a subclass of `FastView`. I thought it was a superclass.